harness/harness-common.rb

```diff
@@ -52,10 +52,11 @@ def get_rss
 yb_env_var = ENV.fetch("RESULT_JSON_PATH", default_path)
 YB_OUTPUT_FILE = File.expand_path yb_env_var
 
-def return_results(times)
+def return_results(times, warmup_iterations)
```

Comment:
```suggestion
def return_results(times, warmup_iterations)
```

This seems like a bit of an awkward setup. It might be better to split the times into two lists: `times` (timing iteration values) and `warmup_itrs` (warmup iteration values) or something like that, instead of passing a number of warmup iterations separately?

Reply:
Yeah I also thought about that, and it's good in that it encourages treating warmup & measurements separately.
It requires quite a bit larger changes, but I can try that if you'd like?
That also means quite some changes in the JSON file, at least the one created by the harness.

There is also the question whether the JSON file created by run_benchmarks.rb should track warmup too, I think it should.

Reply:
All done now